src/Atoms/MainBtn/MainBtnStyled.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import COLORS from '../../Styles/varsStyledComponents';
+import FONTS from '../../Styles/fontsStyledComponents';
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const MainBtnStyled = styled.button`
+  display: block;
+  width: 343px;
+  height: 50px;
+  color: ${(props) =>
+    !props.primary
+      ? !props.secondary
+        ? `${COLORS.grey800}`
+        : `${COLORS.primary601}`
```

Comment:
This becomes very hard to read and follow. If possible, try not to nest ternary operations.

Reply:
Done.  One level ternary operators.